GOVERNANCE.md

```diff
@@ -0,0 +1,136 @@
+### Undici Working Group
+
+The Node.js Undici is governed by a Working Group
```

Comment:
Isn't something missing after "The Node.js Undici" ?

Reply:
Fixed, good spot!